### DISCUSSION TITLE
Make it possible to load multiple custom attacks, and play a 'custom' fight

### DISCUSSION THREAD

**Discussion by wubbzyfan** [Jcw87/c2-sans-fight#146] [Ideas]
Basically, it'd be another option, and from there you could load custom attacks, and then start a fight (after choosing how many dodges until victory) using them as the attacks instead of the base ones, maybe make it possible for custom attacks to have a 'turn' value to always happen on.

**Comment by dl-tg** [comment]
[Answer](https://github.com/Jcw87/c2-sans-fight/issues/145#issuecomment-1473740774)
